tests/integration_tests/test_pytorch_lightning.py

```diff
@@ -111,6 +110,7 @@ def validation_step(  # type: ignore
             accuracy = torch.tensor(0.6)
 
         self.log("accuracy", accuracy, sync_dist=True)
+        return {"validation_accuracy": accuracy}
```

Comment:
[Note] The return value of `validation_step` can be None, but the current change seems reasonable. `Dict[str, Any]` is also expected as can be seen in the [`STEP_OUTPUT` definition](https://github.com/Lightning-AI/lightning/blob/a008801e25a6e93214b410b68ca1edf2e3ddae14/src/pytorch_lightning/utilities/types.py#L35).

https://github.com/Lightning-AI/lightning/blob/a008801e25a6e93214b410b68ca1edf2e3ddae14/src/pytorch_lightning/core/module.py#L764